apps/nextjs/src/components/assistant/assistant-panel.module.css

```
@@ -0,0 +1,299 @@
+    min-height: 32rem;
+    padding-block: 1.5rem;
+  }
+}
+
+@media (max-width: 30em) {
+  .historyButton {
+    max-width: 2.25rem;
+    padding-inline: 0.5rem;
+  }
+
+  .historyButton :global(.mantine-Button-section) {
```

Stylelint (17.14.1)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)